metrics/micrometer/src/test/java/com/datastax/oss/driver/internal/metrics/micrometer/MicrometerNodeMetricUpdaterTest.java

```diff
@@ -190,6 +195,60 @@ public void should_create_timer(
     assertThat(timer.count()).isEqualTo(10);
     HistogramSnapshot snapshot = timer.takeSnapshot();
     assertThat(snapshot.histogramCounts()).hasSize(2);
+    assertThat(snapshot.percentileValues()).hasSize(3);
```

Comment:
nit: it would provide better coverage if you could assert the exact set of percentiles you expect here instead of just the number of items (i.e. `[0.75, 0.95, 0.99]`) - same feedback to the other test in `MicrometerSessionMetricUpdaterTest`

Reply:
added the validation